.github/workflows/node.js.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest #Can change the runtime environment according to needs
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
let's do 14 and 16. Others are not needed :)

Reply:
done :)